pkg/tfgen/parse_markdown_test.go

```diff
@@ -2,15 +2,21 @@ package tfgen
 
 import (
 	"encoding/json"
-	"fmt"
 	"os"
+	"path/filepath"
+	"strconv"
 	"strings"
 	"testing"
 
 	bf "github.com/russross/blackfriday/v2"
 	"github.com/stretchr/testify/assert"
 )
 
+var pulumiAccept = func() bool {
```

Comment:
Why is this not `func pulumiAccept() bool`?

Reply:
`pulumiAccept` is a `bool`, not a `func() bool`. Notice the invocation at the end.